Makefile

```diff
@@ -150,6 +150,7 @@ black-fix:
 # Runs ruff, a linter for python.
 lint:
 	poetry install --only lint --no-root
+	poetry run ruff --version
```

Comment:
ruff doesn't output to stdout on success, so this at least gives an indication of what is going on, and what version is running.